docs/getting-started-guide/master.adoc

```diff
@@ -8,15 +8,15 @@ include::topics/templates/document-attributes.adoc[]
 :context: getting-started-guide
 :getting-started-guide:
 
-= {GettingStartedBookName}
+= {IntroToMTABookName}
```

Comment:
Can you please upload a preview to the intro guide? I don't see it listed in the links.